userguide/storage.rst

```diff
@@ -2488,7 +2476,8 @@ Replication depends on SSH, disks, network, compression, and
 encryption to work. A failure or misconfiguration of any of these can
 prevent successful replication.
 
-Replication logs are located in :file:`var/log/zettarepl.log`.
+All replication logs are located in :file:`/var/log/zettarepl.log`. You can
+inspect logs of any specific replication task by clicking on it's state.
```

Comment:
As above about possessives: no apostrophe in possessive "its", but avoid possessives anyway because they are less clear.

Finally, it's not clear what "state" is referring to here.  Probably should use at least `` :guilabel:`State` `` if it is a UI element.

```suggestion
inspected clicking on their state.
```